cypress/integration/user-journey.js

```diff
@@ -0,0 +1,233 @@
+describe('User journey of login and upload abstracts, visualise then delete upload', () => {
+
+    beforeEach(() => {
+        cy.visit('/');
+        cy.viewport(1920, 1080)
+    });
+
+    it('Lets login, go to results page, check its empty then try to upload an abstract file', () => {
+            cy.visit('/logout');
+            cy.visit('/');
+            cy.get('#side-menu')
+                .contains('Login')
+                .click()
+            cy.get('title')
+                .invoke('text')
+                .should('equal', 'TeMMPo: Login')
+
+            cy.get('#id_username').type(Cypress.env('CREDENTIALS_USR'));
+            cy.get('#id_password').type(Cypress.env('CREDENTIALS_PSW'));
+            cy.get('button[type=submit]').click()
+
+            cy.get('title')
+                .invoke('text')
+                .should('equal', 'TeMMPo: My list')
+
+        // goto results tab and check its empty
+
+            cy.get('#side-menu')
+                .contains('Results')
+                .click()
+
+            cy.get('.page-header')
+                .should('have.text', 'Results')
+
+            cy.get('table')
+                .find('tr')
+                .eq(1)
+                .contains('No data available in table', { matchCase: false })
```

Comment:
@Rodsy This test expects there to be no search results present and also incorporates the test performed by the visualisations tests.  Can you remove the`cypress/integration/visualisations.js` script please?